homeassistant/components/image_processing/__init__.py

```diff
@@ -194,6 +194,7 @@ async def async_update(self) -> None:
         await self.async_process_image(image.content)
 
 
+# pylint: disable-next=hass-enforce-class-module
```

Comment:
Could we modify the parser to allow a component package with the same name as a platform to define a class inheriting the entity? The entity integrations owns the base class and should be allowed to do whatever it wants with that name.

Reply:
Done